docs/sources/administration/configuration.md

```diff
@@ -475,7 +475,7 @@ mitigate the risk of [Clickjacking](https://www.owasp.org/index.php/Clickjacking
 
 ### strict_transport_security
 
-Set to `true` if you want to enable HTTP `Strict-Transport-Security` (HSTS) response header. This is only sent when HTTPS is enabled in this configuration. HSTS tells browsers that the site should only be accessed using HTTPS.
+Set to `true` if you want to enable HTTP `Strict-Transport-Security` (HSTS) response header. Only use this when HTTPS is enabled in your configuration, or when there is another upstream system that ensures your application does HTTPS (like a frontend load balancer). HSTS tells browsers that the site should only be accessed using HTTPS.
```

Comment:
I think it's more effective to just explain the effect of HSTS, to keep the description brief.

@oddlittlebird WDYT?

```suggestion
Set to `true` if you want to enable HTTP `Strict-Transport-Security` (HSTS) response header. HSTS tells browsers that the site should only be accessed using HTTPS.
```

Reply:
I like @sakjur's edit, brief and to the point.

Reply:
Agreed. However, it seems like the "when to use" information might be important to include.